mods/doors/init.lua

```diff
@@ -535,8 +536,9 @@ function doors.register_trapdoor(name, def)
 		return meta:get_string("doors_owner") == pn
 	end
 
-	def.on_rightclick = function(pos, node, clicker)
+	def.on_rightclick = function(pos, node, clicker, itemstack, pointed_thing)
```

Comment:
And since `clicker` is passed along also `node`. Additionally since it is then used in the second commit to pass `node` further so that `get_node` doesn't need to be used. This is exactly, why one should add all the parameters in the first place, because this makes it easier to spot the actual issues better.